pkg/fileutil/embed.go

```diff
@@ -37,6 +40,16 @@ func CopyEmbedAssets(fsys embed.FS, sourceDir string, targetDir string) error {
 				if err != nil {
 					return err
 				}
+				if sigFound {
+					// If the file already exists and has the same content, don't overwrite it.
+					if existingContent, err := os.ReadFile(localPath); err == nil && string(existingContent) == string(content) {
+						continue
+					}
+					// If the file already exists and is excluded, don't overwrite it.
+					if excludedFiles != nil && slices.Contains(excludedFiles, localPath) {
```

Comment:
I didn't know about slices.Contains(). We should really change ArrayContainsString() which I've been using forever to just do this.

Reply:
It's a new feature from go 1.21